docs/manuals/core/entities/orchestrating-and-job-execution.md

```diff
@@ -139,14 +144,95 @@ task for execution. When submitting a task, you can also use the two parameters
     task.submit()
     ```
 
+# Submission
+
+Each time a `Scenario^`, a `Sequence^` or a `Task^` is submitted, a new `Submission^` entity is created.
+
+## Submission attributes
+
+Here is the list of the `Submission^`'s attributes:
+
+- _entity_id_: The identifier of the entity that was submitted.
```

Comment:
```suggestion
- *entity_id*: The identifier of the entity that was submitted.
```
and in the list below - use asterisks instead of underscore to avoid parsing problems with Python identifiers